src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/PerformanceAnalyzerRCAHealthCheckIT.java

```diff
@@ -0,0 +1,64 @@
+package com.amazon.opendistro.elasticsearch.performanceanalyzer;
+
+import com.amazon.opendistro.elasticsearch.performanceanalyzer.util.WaitFor;
+import com.fasterxml.jackson.databind.JsonNode;
+import org.apache.http.HttpStatus;
+import org.apache.http.util.EntityUtils;
+import org.elasticsearch.client.Request;
+import org.elasticsearch.client.Response;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.Iterator;
+import java.util.Objects;
+import java.util.concurrent.TimeUnit;
+
+public class PerformanceAnalyzerRCAHealthCheckIT extends PerformanceAnalyzerIntegTestBase {
+    @Test
+    public void checkMetrics() throws Exception {
+        ensurePaAndRcaEnabled();
+        final String[] jsonString = new String[1];
+        WaitFor.waitFor(() -> {
+            Request request = new Request("GET",
+                    "/_opendistro/_performanceanalyzer/metrics/?metrics=Disk_Utilization&agg=max&dim=&nodes=all");
+            Response resp = paClient.performRequest(request);
+            Assert.assertEquals(HttpStatus.SC_OK, resp.getStatusLine().getStatusCode());
+            jsonString[0] = EntityUtils.toString(resp.getEntity());
+            JsonNode root = mapper.readTree(jsonString[0]);
```

Comment:
This will be very frequently used. Therefore, do you think we should create a helper method in `PerformanceAnalyzerIntegTestBase` that takes URI and request PARAMS and then converts the response to Json and returns ?

Reply:
I agree that this would be very useful and it's much better than ad hoc JSON parsing, but I think that can be a near future iteration and doesn't have to be a part of this PR